src/docs/Contributing/Designers/NamingConvention/index.mdx

```diff
@@ -0,0 +1,125 @@
+
+### Symbol categorisation
+
+We use this categorisation to create consistency in our libraries:
+
+| Categrory             | Usage                                                                                                                                                    |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **👾 For developers** | These symbols should not be used in your designs. They are meant to share the different states of a component with the development team.                 |
+| **🛠 ToolKit**         | This section contains the symbols used to build the final symbols. They are not meant to be used directly but to be accessible through symbol overrides. |
+| **Component**         | This is where you'll find the ready to use symbols of the library.                                                                                       |
+
+<br />
+
+-- Image --
+
+**Tips to create hierarchy:**
+
+- We use numbers as prefixes to order components depending on their importance. For example, `Button / 01-Solid / 01-S / 01- Primary`
+
+-- Image --
+
+- We use a double hyphen as suffixes to indicate a specific information. For example, `Button / 01-Solid / 01-S / 01- Primary--Campus`
+
+-- Image --
+
+### Layers naming
+
+#### Shapes
+
+Please use the following naming conventions for your shapes:
+
+| Type           | Description                                                     |
+| -------------- | --------------------------------------------------------------- |
+| **Frame**      | Frame is the combination of a **background** and an **outline** |
+| **Background** | Background is the fill of your component                        |
+| **Outline**    | Outline is the border of your component                         |
+| **Divider**    | This allows you to break the rhythm of the page                 |
+| **Shadow**     | This allows you to create relief                                |
+
+<br />
+
+<HintItem>
+  Always rename your layers to have an understable file for others.
+</HintItem>
+
+<HintItem dont="true">
+  Never use layer names like rectangle, oval, etc.
+</HintItem>
+
+#### Icons
+
+Please follow these instructions for the naming of your icon in the layer list.
+
+|  Type                 | Icon name                       | Description                                                           |
+| --------------------- | ------------------------------- | --------------------------------------------------------------------- |
+| **Customisable icon** | Choose the icon here 👇         | Use this naming if your icon is overridable                           |
+| **Static icon**       | Icon action (e.g: _Close Icon_) |  If your icon is not overridable please summarise what action it does |
+
+#### Images
+
+Two cases are available for naming images layers.
```

Comment:
unclear, should I name all images either `Picture` or `Illustration` ?

Reply:
I think the problems is with the word `cases` Two `options` / `possibilities` ?